docs/ENVIRONMENT_VARIABLES.md

```diff
@@ -0,0 +1,82 @@
+# Environment Variables
+
+To run the backend you need to export environment variables.
+
+These are the needed environment variables:
+```
+export FLASK_ENVIRONMENT_CONFIG=<dev-or-test-or-prod-or-local>
+export SECRET_KEY=<your-secret-key>
+export SECURITY_PASSWORD_SALT=<your-security-password-salt>
+export MAIL_DEFAULT_SENDER=<mail-default-sender>
+export MAIL_SERVER=<mail-server>
+export APP_MAIL_USERNAME=<app-mail-username>
+export APP_MAIL_PASSWORD=<app-mail-password>
+export MOCK_EMAIL = <True-or-False>
+```
+
+## Environment Variables Description
+
+### Run Configuration
+
+| Environment Variable     | Description                                                                                                                                                                                                                    | Example |
+|--------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+
+| FLASK_ENVIRONMENT_CONFIG | Short running environment name so that Flask know which configuration to load. Currently, there are 4 options for this: `dev`, `test`, `local` and `prod`.  If you want to use a simple local database (e.g.: sqlite) then set your environment config value as `local`. To use the development environment with a remote database configuration you should use `dev` as a value.  | local     |
```

Comment:
![image](https://user-images.githubusercontent.com/11148726/87875982-ae1f0000-c9cc-11ea-8886-bc6d4a52c013.png)

This is not being rendered as a markdown table, can you try to fix this @mtreacy002 ?

Reply:
Ok, will do that now.

Reply:
@isabelcosta . I've fixed thee table. Can you please check if it's ok now? Ta
![Screen Shot 2020-07-20 at 11 17 27 am](https://user-images.githubusercontent.com/29667122/87890669-c6e5ef00-ca7a-11ea-8849-47fb12191805.png)